src/test/java/com/cloudant/tests/CloudantClientHelper.java

```diff
@@ -36,16 +36,17 @@
     public static final String COUCH_PASSWORD;
     public static final String SERVER_URI;
     public static final String SERVER_URI_WITH_USER_INFO;
+    public static final URL SERVER_URL;
```

Comment:
Doesn't this make the `SERVER_URI` obsolete? We can delete the field `SERVER_URI` and just use a local to make `SERVER_URL` to help avoid accidental usage of the wrong one?